### PR TITLE
Edit and add CI tests to run code from docs/examples.

### PR DESCRIPTION
**Pull request by banasraf**
In the tests that are also presented as examples, I copy the relevant code from docs/examples instead of duplicating it. 
I also added a test that runs perf_analyzer example.

### CODE REVIEW

**Review by jantonguirao — DISMISSED** [review]

File: qa/L0_perf_analyzer_example/test.sh [hunk 1]
```diff
@@ -19,28 +21,7 @@
...
+: ${GRPC_ADDR:=${1:-"localhost:8001"}}
 
-parameters: [
-  {
-    key: "num_threads"
-    value: { string_value: "12" }
-  }
-]
+perf_analyzer -m dali -i grpc -u $GRPC_ADDR -b 64 --input-data test_image --shape DALI_INPUT_0:`stat --printf="%s" test_image/DALI_INPUT_0` -f results.txt
+[[ `tail -n1 results.txt` =~ ^1,[0-9]{3}\.[0-9],([0-9]{1,6},){10}[0-9]{1,6}$ ]] && echo "Output Correct"
```
[the remark this answers is not in the document]
Comment (banasraf) [reply qa/L0_perf_analyzer_example/test.sh:28]: Done

File: qa/L0_perf_analyzer_example/setup.sh [hunk 2]
```diff
@@ -19,21 +21,6 @@
...
-]
-
-output [
-  {
-    name: "DALI_OUTPUT_0"
-    data_type: TYPE_UINT8
-    dims: [ -1 ]
-  }
-]
+cp ${DALI_BACKEND_REPO_ROOT}/docs/examples/perf_analyzer/model_repository model_repository/ -r
```
[the remark this answers is not in the document]
Comment (banasraf) [reply qa/L0_perf_analyzer_example/setup.sh]: done

File: qa/L0_inception_ensemble/setup.sh [hunk 3]
```diff
@@ -1,14 +1,27 @@
...
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-wget -O /tmp/inception_v3_2016_08_28_frozen.pb.tar.gz \
-     https://storage.googleapis.com/download.tensorflow.org/models/inception_v3_2016_08_28_frozen.pb.tar.gz
-(cd /tmp && tar xzf inception_v3_2016_08_28_frozen.pb.tar.gz)
-mv /tmp/inception_v3_2016_08_28_frozen.pb inception_graphdef/1/model.graphdef
+cp ${DALI_BACKEND_REPO_ROOT}/docs/examples/inception_ensemble/model_repository model_repository/ -r
```
[the remark this answers is not in the document]
Comment (banasraf) [reply qa/L0_inception_ensemble/setup.sh]: done

File: qa/L0_identity/setup.sh [hunk 4]
```diff
@@ -1,9 +1,27 @@
...
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-mkdir -p dali_identity/1
-python identity_pipeline.py dali_identity/1/model.dali
-echo "Identity model ready."
 
-popd
+cp ${DALI_BACKEND_REPO_ROOT}/docs/examples/identity/model_repository model_repository/ -r
```
[the remark this answers is not in the document]
Comment (banasraf) [reply qa/L0_identity/setup.sh]: done

**Review by jantonguirao — APPROVED** [review]

**Review by szalpal — APPROVED** [review]

**Pull request merged**